tests/openvino/test_decoder.py

```diff
@@ -120,6 +121,8 @@ class OVModelForCausalLMIntegrationTest(unittest.TestCase):
 
     if is_transformers_version(">=", "4.53.0"):
         SUPPORTED_ARCHITECTURES += ("arcee",)
+        if is_openvino_version(">=", "2025.3.0"):
```

Comment:
```suggestion
        if is_openvino_version(">=", "2025.4.0"):
```
Shouldn't it be 2025.4?